website/config.toml

```diff
@@ -113,10 +113,8 @@ algolia_docsearch = false
 
 # User interface configuration
 [params.ui]
-# Enable to show the side bar menu in its compact state.
-sidebar_menu_compact = true
 #  Set to true to disable breadcrumb navigation.
```

Comment:
```suggestion
#  Set to true to disable breadcrumb navigation. 
```

Reply:
add one empty character in the comment to fix the CI because of the migration for the Travis.org